### PR TITLE
validate cpf numbers with letters fix #39

### LINKED ISSUE #39: CPF with letters pass validation
**Describe the bug**
I'm not exactly sure if this is expected behaviour or a bug, but if I try to validade a CPF string with letters in it, the validator returns true IF the numbers match a valid CPF.

**To Reproduce**
```js
// expected return: true
// returns: true
isValidCpf('391.838.380-66')

// expected return: false
// returns: false
isValidCpf('391.838.300-66')

// expected return: false
// returns: false
isValidCpf('391.838.3800-66')

// expected return: false
// returns: true
isValidCpf('foo391.838.380-66')

// expected return: false
// returns: true
isValidCpf('foo391.838.38test0-66')

// expected return: false
// returns: true
isValidCpf('3s9o1.8m3e8.3t8e0s-6t6')

// expected return: false
// returns: false (extra zero at end)
isValidCpf('3s9o1.8m3e8.3t8e0s-6t60')
 ```

While it seems nice that isValidCpf() get only the number types, I think this could lead to problems. I don't know, maybe I should use the /helper-only-numbers as well? 

**Environment:**
 - OS: macOS Mojave v10.14
 - Node: v10.15.0
 -  @brazilian-utils/validators: v0.1.9

### PR DESCRIPTION
**Pull request by fxamauri**

### CODE REVIEW

**Comment by andreoav** [comment]
I think it is possible to update to use a single regex if you make the `.` and the `-` optional.

**Comment by fxamauri** [comment]
> single regex if you make the `.` and the `-` optional.

:+1: I agree, I'm going to make this change.
I will already take advantage of and validate the CNPJ as well.

**Comment by andreoav** [comment]
Awesome.
After you add it to the CNPJ as well I think we are ready to merge.

I will add this to the refactoring branch #36

File: packages/is-valid-cpf/src/index.ts [hunk 1]
```diff
@@ -18,5 +20,8 @@ export default function isValidCpf(cpf: string) {
 
   const numericCPF = onlyNumbers(cpf);
 
-  return isValidLength(numericCPF) && !belongsToBlacklist(numericCPF) && isValidChecksum(numericCPF);
+  return isValidCpfFormat(cpf)
+    && isValidLength(numericCPF)
```
Comment (marcelolx) [inline packages/is-valid-cpf/src/index.ts]: Same case here.

File: packages/is-valid-cnpj/src/index.ts [hunk 2]
```diff
@@ -24,5 +26,8 @@ export default function isValidCnpj(cnpj: string) {
 
   const numericCNPJ = onlyNumbers(cnpj);
 
-  return isValidLength(numericCNPJ) && !belongsToBlacklist(numericCNPJ) && isValidChecksum(numericCNPJ);
+  return isValidCnpjFormat(cnpj)
+    && isValidLength(numericCNPJ)
```
Comment (marcelolx) [inline packages/is-valid-cnpj/src/index.ts]: You are validate format of cnpj with regex, why do you need isValidLength? Regex not validate also length of cnpj?

Comment (fxamauri) [reply packages/is-valid-cnpj/src/index.ts]: You're right, not necessary to validate the size, I'll fix it.

**Pull request merged**